[2020-10-07 17:51:07,789][root][INFO] STARTED LOGGING: logs/eeBookCalendarPrices_TestSuite_192.168.225.15
[2020-10-07 17:51:08,389][root][DEBUG] overriding grid, using local browser
[2020-10-07 17:51:11,257][root][INFO] Running case: TC#33
[2020-10-07 17:51:11,436][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=POS&destinCode=JFK&numAdt=1&numChd=1&numInf=1&numJun=1&departure=2020-10-27&appid=4
[2020-10-07 17:51:17,960][root][INFO] ***Looking for priced OUTBOUND flights in the previous 5 days***
[2020-10-07 17:51:21,489][root][INFO] ***Found 7 valid priced dates in the outbound flight calendar***
[2020-10-07 17:51:29,070][root][INFO] No previously active price and date saved yet.
[2020-10-07 17:51:29,070][root][INFO] Lowest outbound active price is correctly selected by default: 299.07
[2020-10-07 17:51:29,070][root][INFO] Saving the new outbound active price and date: WED 21 OCT 299.07 USD
[2020-10-07 17:51:36,773][root][INFO] Previously active calendar price and date are displayed correctly: WED 21 OCT 299.07
[2020-10-07 17:51:36,773][root][INFO] Lowest outbound active price is correctly selected by default: 299.07
[2020-10-07 17:51:36,773][root][INFO] Saving the new outbound active price and date: THU 22 OCT 299.07 USD
[2020-10-07 17:51:39,530][root][INFO] WARNING: Couldn't open an outbound priced calendar date!
[2020-10-07 17:51:39,531][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/element'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,532][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129560580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/element
[2020-10-07 17:51:39,532][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/element'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,533][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129509d90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/element
[2020-10-07 17:51:39,533][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/element'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,533][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129509e20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/element
[2020-10-07 17:51:39,534][root][DEBUG] FOLDER CREATION: ./screenshots/ returned an error: [Errno 17] File exists: './screenshots/'
[2020-10-07 17:51:39,535][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/window/maximize'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,535][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129509250>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/window/maximize
[2020-10-07 17:51:39,535][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/window/maximize'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,535][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129505760>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/window/maximize
[2020-10-07 17:51:39,535][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/window/maximize'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,536][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129505940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/window/maximize
[2020-10-07 17:51:39,536][root][INFO] Running case: TC#34
[2020-10-07 17:51:39,696][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=POS&destinCode=JFK&numAdt=1&numChd=1&numInf=1&numJun=1&departure=2020-10-24&return=2020-11-03&appid=4
[2020-10-07 17:51:39,697][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,697][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129505e50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,697][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,698][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129520f70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,698][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,698][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1295347c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,698][root][INFO] Running case: TC#35
[2020-10-07 17:51:39,857][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=SLU&destinCode=JFK&numAdt=1&numChd=1&numInf=1&numJun=1&departure=2020-10-27&appid=4
[2020-10-07 17:51:39,858][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,858][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1295343a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,859][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,859][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129534e50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,860][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:39,860][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129534280>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:39,861][root][INFO] Running case: TC#36
[2020-10-07 17:51:40,024][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=SLU&destinCode=JFK&numAdt=1&numChd=1&numInf=1&numJun=1&departure=2020-10-24&return=2020-11-01&appid=4
[2020-10-07 17:51:40,025][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,026][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1295404c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,027][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,028][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d0d30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,029][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,029][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129540730>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,030][root][INFO] Running case: TC#37
[2020-10-07 17:51:40,197][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=POS&destinCode=JFK&numAdt=2&numChd=3&numInf=2&numJun=2&departure=2020-10-24&appid=4
[2020-10-07 17:51:40,198][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,199][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129540c10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,200][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,200][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129520a60>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,201][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,202][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d0c10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,203][root][INFO] Running case: TC#38
[2020-10-07 17:51:40,374][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=POS&destinCode=JFK&numAdt=2&numChd=2&numInf=2&numJun=2&departure=2020-10-25&return=2020-11-01&appid=4
[2020-10-07 17:51:40,376][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,376][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d3c10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,377][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,378][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d3670>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,379][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,379][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d3910>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,380][root][INFO] Running case: TC#39
[2020-10-07 17:51:40,547][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=SLU&destinCode=JFK&numAdt=2&numChd=2&numInf=2&numJun=2&departure=2020-10-25&appid=4
[2020-10-07 17:51:40,549][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,549][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d3190>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,550][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,550][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129534c40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,552][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,552][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1295202e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,553][root][INFO] Running case: TC#40
[2020-10-07 17:51:40,720][root][INFO] https://bwaweb-eebkg-int.2e-systems.com/en/fb/availability?originCode=SLU&destinCode=JFK&numAdt=3&numChd=2&numInf=3&numJun=3&departure=2020-10-24&return=2020-11-05&appid=4
[2020-10-07 17:51:40,722][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,722][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe129520790>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,723][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,724][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1295056a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:40,725][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b/url'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:40,725][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d0d90>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b/url
[2020-10-07 17:51:42,729][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:42,729][urllib3.connectionpool][WARNING] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d0df0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b
[2020-10-07 17:51:42,729][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:42,729][urllib3.connectionpool][WARNING] Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294d38b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b
[2020-10-07 17:51:42,730][urllib3.util.retry][DEBUG] Incremented Retry for (url='/session/0e182460675f66c670017033adb0e40b'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
[2020-10-07 17:51:42,730][urllib3.connectionpool][WARNING] Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe1294eeaf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0e182460675f66c670017033adb0e40b
